### PR TITLE
Updating jackson data bind

### PR DESCRIPTION
**Pull request by jambo1**
## What does this change?

Updating data bind

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -34,7 +34,7 @@ libraryDependencies ++= Seq(
   "org.mockito" % "mockito-all" % "1.9.5" % "test",
   
   // All the below are required to force aws libraries to use the latest version of jackson
-  "com.fasterxml.jackson.core" % "jackson-databind" % jacksonVersion,
+  "com.fasterxml.jackson.core" % "jackson-databind" % "2.10.5.1",
```
Comment (kelvin-chappell) [inline build.sbt:37]: Should all the Jackson libraries be on the same version?

Comment (jambo1) [reply build.sbt:37]: Ideally, I tried updating them all and ran into some issues so tried only the one

**Review by kelvin-chappell — APPROVED** [review]
👍

**Comment by jambo1** [comment]
Redundant thanks to #23

**Pull request closed without merging**